src/main/java/core/basesyntax/models/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class Rectangle implements Figure {

```

Comment:
Don't begin class implementation with an empty line. Please remove the redundant empty line before the class declaration.